webclient/app/src/app/features/decision/DecisionOverview.jsx

```diff
@@ -30,6 +30,7 @@ function DecisionOverview() {
     const [rowData, setRowData] = React.useState({});
     const [automaticNext, setAutomaticNext] = React.useState(false);
     const locale = i18n.language == "de" ? deDE : enUS
+    const pageSize = 10;
```

Comment:
10 seems very small... wouldn't we at least want to fill a reasonably sized screen?